src/components/Calculator.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Linters are great tools that enable you to flag programming errors. We are not supposed to change their rules by disabling them. Kindly resolve the issues raise by linters. Do not disable them.
- **Hint:** I guess you experiencing linters issues because of the class-based component :smiley: . I'll advise you to extend React.PureComponent to resolve the linters issue that's class `Calculator extends React.PureComponent {`
[Readmore... ](https://www.geeksforgeeks.org/reactjs-pure-components/)